dm/ctl/ctl.go

```diff
@@ -72,7 +72,7 @@ func NewRootCmd() *cobra.Command {
 		master.NewOperateLeaderCmd(),
 		master.NewListMemberCmd(),
 		master.NewOperateSchemaCmd(),
-		master.NewGetTaskCfgCmd(),
```

Comment:
I'm afraid some users have remembered this command, could we keep it as an alias of `get-config` or give redirect message?

Reply:
alias it for both interactionMode and commandMode